pkg/migrations/op_add_column.go

```diff
@@ -78,16 +78,20 @@ func (o *OpAddColumn) Start(ctx context.Context, conn db.DB, latestSchema string
 
 	var tableToBackfill *schema.Table
 	if o.Up != "" {
-		err := createTrigger(ctx, conn, triggerConfig{
-			Name:           TriggerName(o.Table, o.Column.Name),
-			Direction:      TriggerDirectionUp,
-			Columns:        table.Columns,
-			SchemaName:     s.Name,
-			LatestSchema:   latestSchema,
-			TableName:      table.Name,
-			PhysicalColumn: TemporaryName(o.Column.Name),
-			SQL:            o.Up,
-		})
+		createUpTrigger := NewCreateTriggerAction(
+			conn,
```

Comment:
nit: Could put `conn` on the same line as `NewCreateTriggerAction` here and everywhere else; makes things a bit more readable IMO.